src/js/view/controls/controls.js

```diff
@@ -336,6 +361,10 @@ export default class Controls {
 
         // Hide controls when focus leaves the player
         const blurCallback = (evt) => {
+            //  Remove new shortcut tooltip after first read.
+            if (!OS.mobile) {
+                this.playerContainer.removeAttribute('aria-describedby');
```

Comment:
i don't think we should remove this. If you're worried about screen readers announcing it multiple times, we could look into using aria-live or alternatives

Reply:
It's my understanding that they want screen readers to read that description when focusing on the player the first time (in the non-persistence sense of "first time"). It has to be removed here to prevent it from occurring every time the player is focused.

Reply:
is there danger in a broad brush removal of this attribute on the player container? are there other elements that we may want to repeat the screen-read on focus for? given it's a shared component, that is.